docker-compose.yml

```diff
@@ -27,7 +32,7 @@ services:
       - --api
     volumes:
       - "/var/run/docker.sock:/var/run/docker.sock:ro"
-      - "./etc:/etc/traefik"
+      - "./etc/conf.d:/etc/traefik/conf.d"
```

Comment:
Since there can be multiple configs as we add more services, let’s use ./etc/traefik/conf.d

Reply:
i was thinking to use a separate dir for the acme certs (you had an ./etc/acme directory before) and keeping this dir free for traefik specific config files

Reply:
Ah good idea, you can specify the path to acme certs separately.   Then put certs into /etc/traefik/acme And make it obvious there are two directories for traefik on the host side?